packages/app-store/alby/lib/PaymentService.ts

```diff
@@ -84,6 +87,7 @@ export class PaymentService implements IAbstractPaymentService {
       }
       return paymentData;
     } catch (error) {
+      console.error(error);
       log.error("Alby: Payment could not be created", bookingId);
```

Comment:
@ThyMinimalDev how about this instead?

`log.error("Alby: Payment could not be created", bookingId,  JSON.stringify(error));
`